File: 코딩 기초 트레이닝/글자 지우기-p181900/글자 지우기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181900

function solution(my_string, indices) {
    var answer = '';
    my_string = my_string.split("");
    my_string.forEach((m, index) => {
        if(!indices.includes(index)) {
            answer += m;
        }
    });

    return answer;
}

File: 코딩 기초 트레이닝/글자 지우기-p181900/글자 지우기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181900

def solution(my_string, indices):
    answer = ''
    my_string = list(my_string)

    for index, s in enumerate(my_string):
        if not index in indices:
            answer += s

    return answer
